6201484321_f1a88ca2cb_b_archive.torrent

```diff
@@ -0,0 +1,5 @@
+d8:announce36:http://bt1.archive.org:6969/announce13:announce-listll36:http://bt1.archive.org:6969/announceel36:http://bt2.archive.org:6969/announceee7:comment650:This content hosted at the Internet Archive at https://archive.org/details/6201484321_f1a88ca2cb_b
```

Comment:
I don't actually remember what is in this torrent. But this is the torrent meta info file. Also, more part of an integration test.